meetup_ballot/ballot.py

```diff
@@ -88,7 +89,17 @@ def does_member_name_looks_like_spam(member_name):
     return False
 
 
-def filter_spam_members(member_ids, client):
+def read_name_exceptions(name_exceptions_csv):
+    """ Read member names from csv files"""
+    member_ids = []
+    with open(name_exceptions_csv, newline='') as name_exceptions:
```

Comment:
Is there a specific reason to disable translation of newlines?

Reply:
Not really, I will remove it.